devel/gdb_load_file

```diff
@@ -0,0 +1,13 @@
+target remote localhost:2331
```

Comment:
this is zedboard-specific, should probably be a part of phoenix-rtos-project for zedboard (to be also available when You're developing natively, not through docker)

Reply:
What location will be good? `_projects/armv7a9-zynq7000-zedboard/files/gdb_load_file`?